src/index.ts

```diff
@@ -53,6 +53,7 @@ export interface PluginClass {
   ) => void
   willShowExperience?: (host: Ketch, config: Configuration, type: string) => void
   experienceHidden?: (host: Ketch, config: Configuration, reason: string) => void
+  getIdentities?: () => Promise<Identities>
```

Comment:
rely on init host and config

Reply:
Not sure if this is actually advantageous vs just sending - thoughts?

Reply:
negative. this only affects class plugins. this should be on the host. `registerIdentityProvider(provider: () => Promise<Identities>)`